### DISCUSSION TITLE
Performance of the new Segmentation model

### DISCUSSION THREAD

**Discussion by omarsayed7** [pyannote/pyannote-audio#1491] [General]
Hello all,
 I measured the DER on custom dataset of the new Segmentation model with WeSpeaker_ResNet34_LM  variant and got 19.4%, while the old Segmentation model from v2.1.1 with WeSpeaker_ResNet34_LM  variant and got 16.9%. 
Could you please explain why the new Segmentation model gets bad results compared to the old one?

Regards,
Omar

**Comment by hbredin** [comment]
Marking this as `cannot_reproduce` until you provide some example files (and corresponding manual annotations) where this happens.
